### PR TITLE
Fixed #35179 -- Made admindocs detect positional/keyword-only arguments.

### PR DESCRIPTION
**Pull request by salvo-polizzi**
Fix [ticket 35179](https://code.djangoproject.com/ticket/35179)

### CODE REVIEW

File: django/utils/inspect.py [hunk 1]
```diff
@@ -56,14 +56,22 @@ def get_func_full_args(func):
...
 def func_accepts_var_args(func):
     """
     Return True if function 'func' accepts positional arguments *args.
     """
-    return any(p for p in _get_callable_parameters(func) if p.kind == p.VAR_POSITIONAL)
+    return any(
+        p
+        for p in _get_callable_parameters(func)
+        if p.kind == p.VAR_POSITIONAL or p.kind == p.POSITIONAL_ONLY
+    )
```
Comment (shangxiao) [inline django/utils/inspect.py:74]: I'm sorry but it appears this patch breaks these 2 functions.

`func_accepts_var_args()` should only return `True` for `VAR_POSITIONAL` - the name of the function and docstrings are clear on this 😊

as for `func_accepts_kwargs()` - the name is a misnomer but the docstrings state the intended purpose. If you look at its usage throughout the codebase though the intention that it only return `True` for `**kwargs`.

**Review by shangxiao — CHANGES_REQUESTED** [review]
Hi thanks for the patch though I doubt this is the correct approach 🤷

File: django/utils/inspect.py (hunk 1 above)
Comment (salvo-polizzi) [reply django/utils/inspect.py:74]: Thanks. I had to read better before

**Comment by shangxiao** [comment]
@salvo-polizzi thanks 🏆

**Pull request merged**